mParticle-Apple-SDK/Kits/MPKitContainer.mm

```diff
@@ -1892,6 +1892,21 @@ - (void)project:(id<MPExtensionKitProtocol>)kitRegister event:(MPEvent *const)ev
     completionHandler(projectedEvents, appliedProjections);
 }
 
+- (nullable NSArray<NSNumber *> *)configuredKitsRegistry {
+    BOOL anyKitsIncluded = self.supportedKits != nil && self.supportedKits.count > 0;
```

Comment:
Sorry for the nonstop nitpicks haha, but the nil checks are unnecessary here. In Objective-C, it's completely valid to send messages (call methods) on nil objects. If they have an object return type, they will return nil, and if they have an integer return type will return 0, etc. 

So in this case, `self.supportedKits.count` will evaluate to 0 if `supportedKits` is nil, so the single check of `BOOL anyKitsIncluded = self.supportedKits.count > 0;` is valid in all cases, even if `supportedKits` is nil.

Same goes for the next line as well, it's cleaner and totally valid to just write: `BOOL self.kitConfigurations.count > 0;`

Reply:
dude, no problem at all. I'd rather learn it hear than go through life writing sub-par obj-c :)